### PR TITLE
Refactor and Miscellaneous Changes

### PR DESCRIPTION
**Pull request by D4isDAVID**
The main goal of this PR is to make coding less miserable.

This includes:
* Consistent indentation (0f528d1) (2d05091)
* Consistent semicolons (07cd21d)
* Organized imports (3fb12bc)
* Using `async` and `await` over `Promise#then()` to reduce callback hell (55e2381) (5bc19b2)
* Other small changes

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -286,86 +296,89 @@ const interactions = {
...
-		})
+			]
+		))[0];
+		if(res.length == 0) {
+			interaction.reply({content: "Failed to find the server.. tf", ephemeral: true});
+			return;
+		}
+		const channelData = res[0];
+
+		let packet;
```
Comment (D4isDAVID) [inline index.js:328]: Possible ugly solution

File: index.js [hunk 2]
```diff
@@ -286,86 +296,89 @@ const interactions = {
...
+			}
+		).then(async () => {
+			packet = await database.execute("UPDATE `channels` SET `votes` = `votes` + 1 WHERE `id` = ?", [channelData.id]).catch(err => {
+				console.error(err);
+			});
+		}).catch(err => {
+			interaction.reply({content: "Failed to vote, The error has been logged to the console.", ephemeral: true});
+			console.error(err);
+		});
+		if (!packet) return;
```
Comment (D4isDAVID) [inline index.js:344]: Possibly ugly solution

**Review by Elior2739 — CHANGES_REQUESTED** [review]
In lines: 143, 201, 277, 494 there are no needed `await`

File: index.js [hunk 3]
```diff
@@ -127,28 +127,26 @@ const commands = [
...
+			}).catch(err => {
+				interaction.editReply({content: "Error happened while creating category, The error has been logged to the console"});
+				console.error(err);
+				return;
+			});
+			await database.execute("INSERT INTO `categories` (`channel`) VALUES (?)", [category.id]).catch(err => {
+				category.delete().catch(err => err); // fuck this
+				interaction.editReply({content: "Failed to insert the category to database, The error has been logged to the console."});
+				console.error(err);
+				return;
```
[the remark this answers is not in the document]
Comment (D4isDAVID) [reply index.js]: Resolved in 5bc19b2

**Comment by D4isDAVID** [comment]
> In lines: 143, 201, 277, 494 there are no needed `await`

Fixed in 903de18

**Pull request merged**